nx_parallel/algorithms/centrality/betweenness.py

```diff
@@ -58,6 +58,9 @@ def betweenness_centrality(
         for chunk in node_chunks
     )
 
+    if not bt_cs:
```

Comment:
I think we should probably put this right after we update `G`.  For this one, that is line 43. We avoid extra work by checking `len(G) == 0` and returning `{}` in that case.  Thoughts?